docs/semgrep-appsec-platform/dashboard-beta.md

```diff
@@ -0,0 +1,135 @@
+---
+slug: dashboard-beta
+title: Dashboard (beta)
+hide_title: true
+description: Use the Semgrep dashboard (beta) to gain an overview of your organization's security posture, including the deployment of guardrails.
+tags:
+  - Semgrep AppSec Platform
+  - Secure guardrails
+---
+
+# Dashboard (beta)
+
+The Semgrep dashboard (beta) is an overview of your organization’s security posture from data aggregated within Semgrep AppSec Platform. It helps you achieve the following goals:
```

Comment:
do we think it's necessary to call out the differences between this and the current dashboard? or what this offers vs the current dashboard? this may not be the right place, but i think it's an importance resource for us to have cc @milanwilliams

Reply:
I wrote out which parts are exclusive to the beta in the beta callout. I didn't make it too long because that's more or less the content of the doc and would be redundant. 
![image](https://github.com/user-attachments/assets/38e48dad-7a10-4570-8422-46e1fe6c0035)